addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/IPBridgeHandler.java

```diff
@@ -312,22 +314,30 @@ private void parseUpdates() {
                     continue;
                 }
 
-                Integer integrationId = new Integer(matcher.group(2));
+                Integer integrationId;
+
+                try {
+                    integrationId = new Integer(matcher.group(2));
+                } catch (NumberFormatException e1) {
+                    logger.warn("Integer conversion error parsing update: {}", line);
+                    continue;
+                }
+                paramString = matcher.group(3);
+
+                // Now dispatch update to the proper thing handler
                 LutronHandler handler = findThingHandler(integrationId);
 
                 if (handler != null) {
-                    String paramString = matcher.group(3);
-
                     try {
                         handler.handleUpdate(type, paramString.split(","));
                     } catch (Exception e) {
-                        this.logger.error("Error processing update", e);
+                        logger.error("Error processing update", e);
                     }
                 } else {
-                    this.logger.info("No thing configured for integration ID {}", integrationId);
+                    logger.info("No thing configured for integration ID {}", integrationId);
```

Comment:
This more looks like a warning? If it occurs often and is properly handled it should be reduced to debug.

Reply:
Well, this isn't really my code.  I just stripped "this." off of all logger calls en masse.  But yes, I do think this once can be reduced to debug, since in some cases you may not want to configure things for all the active integration IDs.